sycl/plugins/hip/CMakeLists.txt

```diff
@@ -10,22 +10,18 @@ set(SYCL_BUILD_PI_HIP_HSA_INCLUDE_DIR "/opt/rocm/hsa/include" CACHE STRING "HSA
 set(HIP_HEADERS "${SYCL_BUILD_PI_HIP_INCLUDE_DIR};${SYCL_BUILD_PI_HIP_HSA_INCLUDE_DIR}")
 
 # Create pi_hip library
-add_library(pi_hip SHARED
-  "${sycl_inc_dir}/CL/sycl/detail/pi.h"
-  "${sycl_inc_dir}/CL/sycl/detail/pi.hpp"
-  "pi_hip.hpp"
-  "pi_hip.cpp"
+add_sycl_plugin(hip
+  SOURCES
+    "${sycl_inc_dir}/CL/sycl/detail/pi.h"
+    "${sycl_inc_dir}/CL/sycl/detail/pi.hpp"
+    "pi_hip.hpp"
+    "pi_hip.cpp"
+  INCLUDE_DIRS
+    ${sycl_plugin_dir}
+  LIBRARIES
+    OpenCL-Headers
```

Comment:
I just noticed that each and every plug-in depends on OpenCL-Headers library, which we can move to add_sycl_plugin as well.
Eventually we should remove dependency on OpenCL headers and use only PI declarations.